### PR TITLE
Update services version num

### PR DESCRIPTION
**Pull request by kimbor**
Signed-off-by: Kim Rader <kimrader@gmail.com>

**Description**:
Update platform version and services version for new release

### CODE REVIEW

**Review by nathanklick — COMMENTED** [review]
I think the platform version update may be missing.

**Comment by kimbor** [comment]
> I think the platform version update may be missing.

Yep, I fixed it.

File: hedera-node/hedera-mono-service/src/jmh/java/com/hedera/services/setup/InfrastructureManager.java [hunk 1]
```diff
@@ -36,7 +33,6 @@
 
 public class InfrastructureManager {
     private static final String BASE_STORAGE_DIR = "databases";
-    public static final Cryptography CRYPTO = new CryptoEngine(getStaticThreadManager());
```
Comment (mhess-swl) [inline hedera-node/hedera-mono-service/src/jmh/java/com/hedera/services/setup/InfrastructureManager.java:39]: This wasn't used

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Review by Neeharika-Sompalli — APPROVED** [review]

**Review by qnswirlds — APPROVED** [review]

**Pull request merged**